tests/cross/src/test/scala/tests/pc/CompletionArgSuite.scala

```diff
@@ -31,7 +31,12 @@ class CompletionArgSuite extends BaseCompletionSuite {
        |Main arg1
        |:: scala.collection.immutable
        |""".stripMargin,
-    topLines = Option(3)
+    topLines = Option(3),
+    compat = Map(
+      "3.0" ->
```

Comment:
This can be removed since we ignore the test.

Reply:
Actually, this `compat` is for test `arg1` which is not ignored.
(I'm also confused because GitHub folded the code  😅 )

Reply:
Ach! Sorry about that!